components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -72,7 +72,7 @@ Comment getComment(ApiTypeWrapper apiTypeWrapper, String commentId, Integer repl
      * @return Comments
      * @throws APIManagementException if failed to get comments for identifier
      */
-    CommentList getComments(ApiTypeWrapper apiTypeWrapper, String parentCommentID, Integer replyLimit, Integer replyOffset) throws APIManagementException;
+    CommentList getComments(String apiTypeWrapper, String parentCommentID, Integer replyLimit, Integer replyOffset) throws APIManagementException;
```

Comment:
variable name should be uuid

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10614/commits/b561990adfc350c2aec76fc8e72c88c6ad896f15